pkg/server/app/models.go

```diff
@@ -237,3 +249,33 @@ func unsetEnvVars(app *App, evs []string) {
 		}
 	}
 }
+
+func newListResponse(list []*List) *appb.ListResponse {
+	if list == nil {
+		return nil
+	}
+
+	appNames := []*appb.ListResponse_App{}
+
+	for _, item := range list {
+		if item == nil {
+			continue
+		}
+		var tmp []string
+		for _, elt := range item.Addresses {
+			tmp = append(tmp, elt.Hostname)
```

Comment:
I realized that behavior was fixed at teresa-api/pkg/server/k8s/client.go in the `AddressList`
```
func (k *k8sClient) AddressList(namespace string) ([]*app.Address, error) {
	srvs, err := k.kc.CoreV1().Services(namespace).List(k8sv1.ListOptions{})
	if err != nil {
		return nil, errors.Wrap(err, "get addr list failed")
	}

	addrs := []*app.Address{}
	for _, srv := range srvs.Items {
		for _, i := range srv.Status.LoadBalancer.Ingress {
			h := i.Hostname
			if h == "" {
				h = i.IP
			}
			addrs = append(addrs, &app.Address{Hostname: h})
		}
	}
```